kmip/core/attributes.py

```diff
@@ -154,20 +153,17 @@ def create(cls, name_value, name_type):
 # 3.3
```

Comment:
Given that there are multiple versions of the KMIP spec, I would suggest we specifically state what version of the spec this applies to, along with the chapter. Something along the lines of:

"KMIP Spec 1.1 - 3.3"